test/unit/auth/auth-api-request.spec.ts

```diff
@@ -2515,7 +2515,7 @@ AUTH_REQUEST_HANDLER_TESTS.forEach((handler) => {
 
     describe('getOAuthIdpConfig()', () => {
       const providerId = 'oidc.provider';
-      const path = handler.path('v2beta1', `/oauthIdpConfigs/${providerId}`, 'project_id');
```

Comment:
We also want to rename the APIs to be consistent with the SAML counterparts here and elsewhere.
`oauthIdpConfigs` -> `inboundOauthConfigs`
Check the proto file for confirmation.

Reply:
Seems like oauthIdpConfigs was kept for OnePlatform compliancy.